bcda/client/bluebutton.go

```diff
@@ -184,6 +182,29 @@ func addRequestHeaders(req *http.Request, reqID uuid.UUID) {
 	req.Header.Add("BlueButton-BackendCall", "")
 }
 
+func (bbc *BlueButtonClient) tryRequest(req *http.Request, jobID string) (string, error) {
+	go logRequest(req, jobID)
+	resp, err := bbc.httpClient.Do(req)
+	if resp != nil {
+		defer resp.Body.Close()
+		logResponse(req, resp, jobID)
```

Comment:
Do we want to continue logging the req in the logResponse method even though we just logged it a few lines earlier? The usage of the request parameter in both of those methods is kinda identical. But that could be worked in another ticket.

Reply:
Good question. As long as we have the request/query UUID, we could get the timestamp, URL, and job ID from the request log entry. So it could just be query ID and response code... but maybe we want all the fields for easier searching?